src/test/scala/com/github/saurfang/sas/spark/DefaultSourceSuite.scala

```diff
@@ -1,39 +0,0 @@
-package com.github.saurfang.sas.spark
-
-import org.apache.spark.SharedSparkContext
-import org.apache.spark.sql.SQLContext
-import org.scalatest.{FunSuite, Matchers}
-
-class DefaultSourceSuite extends FunSuite with Matchers with SharedSparkContext {
-  test("Load data via SQL should be the same as via Spark") {
-    val datetimeFile = getClass.getResource("/datetime.sas7bdat").getPath
-
-    val sqlContext = new SQLContext(sc)
-
-    import com.github.saurfang.sas.spark._
-    val dtDF = sqlContext.sasFile(datetimeFile)
-
-    import sqlContext._
-    sql(
-      s"""
-        |CREATE TEMPORARY TABLE dtTable
-        |USING com.github.saurfang.sas.spark
-        |OPTIONS (path "$datetimeFile")
-      """.stripMargin)
```

Comment:
added this back...this test kept breaking on my system (and still does) probably something to do with windows and or my JVM settings :)